pages/partials/questionFooter.ejs

```diff
@@ -71,27 +67,17 @@
                 </button>
               <% } %>
               <% if (showGradeButton) { %>
-                <button class="btn btn-primary question-grade disable-on-submit order-1 mr-1">Save & Grade</button>
-            <%  let attemptsLeft = 0, attemptsTotal = 0, suffix = '';
-                if (question_context == 'student_homework') {
-                    suffix = ' for this version';
-                    attemptsLeft = assessment_question.tries_per_variant - variant.num_tries;
-                    attemptsTotal = assessment_question.tries_per_variant;
-                } else if (question_context == 'student_exam') {
-                    attemptsLeft = instance_question.points_list.length;
-                    attemptsTotal = instance_question.points_list_original.length;
-                }
-                if (attemptsTotal > 0) { %>
-                <span class="badge badge-info ml-1 order-3">
-                    <% if (attemptsLeft > 1) { %>
-                    <%= attemptsLeft %> grade attempts left <%= suffix %>
-                    <% } else if (attemptsLeft == 1 && attemptsTotal > 1) { %>
-                    Last grade attempt <%= suffix %>
-                    <% } else if (attemptsLeft == 1 && attemptsTotal == 1 && question_context == 'student_exam') { %>
-                    Single grade attempt <%= suffix %>
-                    <% } %>
-                </span>
-                <% } %>
+                <button class="btn btn-primary question-grade disable-on-submit order-1 mr-1">Save & Grade
+            <% if (variantAttemptsTotal > 0) { %>
+               <% if (variantAttemptsLeft > 1) { %>
+               <small class="font-italic ml-2"><%= variantAttemptsLeft %> attempts left</small>
+               <% } else if (variantAttemptsLeft == 1 && variantAttemptsTotal > 1) { %>
+               <small class="font-italic ml-2">Last attempt</small>
+               <% } else if (variantAttemptsLeft == 1) { %>
+               <small class="font-italic ml-2">Single attempt</small>
+               <% } %>
```

Comment:
If this is a `singleVariant` question, should we add text here to say `Unlimited attempts`? Otherwise I think it's not entirely clear what will happen for `singleVariant` questions, because we don't display any guidance at all.

Reply:
I think our principle should be:
1. _always_ have help text inside the button saying how many attempts are allowed (at the current variant)
2. If the question allows multiple variants, have the extra text inside the question footer explaining this.

I think the only case we are missing at the moment is `singleVariant` questions on Homeworks?

Reply:
What about instructor view?